gprofiler/profilers/java.py

```diff
@@ -273,11 +302,91 @@ def make_application_metadata(self, process: Process) -> Dict[str, Any]:
         # that loads other libs.
         libjvm_elfid = get_mapped_dso_elf_id(process, "/libjvm")
 
-        metadata = {"java_version": version, "libjvm_elfid": libjvm_elfid}
+        try:
+            jvm_flags = self.get_jvm_flags_serialized(process)
+        except CalledProcessError as e:
+            jvm_flags = [{"error": e.stderr}]
```

Comment:
I thought it's nicer to return an error string like I did for the `exe` and `execfn` fields. No need for it to be a list.

Reply:
Just thought it would be nicer, so if we parse it in the backend or ui we won't need to check the type and rather assume jvm_flags is a list

Reply:
We will still need special handling for the error case - there is only one entry which is a dict of one key == `error`. Doesn't help much. It will be easier to check if it's a string that startswith error lol. Make it a string to be consistent with other sites.